### PR TITLE
[Modules] Support for inGuest Patch support with Azure Update Manager

### PR DESCRIPTION
**Pull request by jdrepo**
# Description

Based on #3954 

</details>

Support for the "installPatches" property in the Microsoft.Maintenance/maintenanceConfigurations resource

### Pipeline references

| Pipeline |
| - |
|[Maintenance - MaintenanceConfigurations](https://github.com/jdrepo/ResourceModules/actions/workflows/ms.maintenance.maintenanceconfigurations.yml) |

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

**Review by AlexanderSehr — CHANGES_REQUESTED** [review]
Looks great. 2 asks though:
- Please also update the readme using our Set-ModuleReadme utility
- If possible, it would also be great if we could add the usage of the parameter to the common test case (which would also need a subsequent update of the Readme). This helps us to make sure the change works and further is a good reference for the community on how the object must look like

**Comment by jdrepo** [comment]
Added missing module README.md and modified common test case. Can you have a look please ?

Another common question: How can I assign the follwong labels to PR:

-  [cat] modules 
- enhancement

**Comment by AlexanderSehr** [comment]
> Added missing module README.md and modified common test case. Can you have a look please ?
> 
> Another common question: How can I assign the follwong labels to PR:
> 
> * [cat] modules
> * enhancement

You don't, but I just did. Will review shortly :)

File: modules/maintenance/maintenance-configuration/.test/common/main.test.bicep [hunk 1]
```diff
@@ -65,15 +67,32 @@ module testDeployment '../../main.bicep' = {
...
-      duration: '05:00'
+      duration: '03:00'
       expirationDateTime: '9999-12-31 23:59:59'
       recurEvery: 'Day'
       startDateTime: '2022-12-31 13:00'
       timeZone: 'W. Europe Standard Time'
     }
     namespace: '${serviceShort}ns'
     visibility: 'Custom'
+
```
Comment (AlexanderSehr) [inline modules/maintenance/maintenance-configuration/.test/common/main.test.bicep:80]:
```suggestion
```

**Comment by AlexanderSehr** [comment]
Looks great. Just 1 comment, then we can merge. Thanks @jdrepo

**Review by AlexanderSehr — APPROVED** [review]

**Comment by AlexanderSehr** [comment]
Here we go. Thanks @jdrepo

**Comment by jdrepo** [comment]
Thanks. 
What will happen to the referenced issue #3954  in the Product backlog ? 
Will it be closed automatically or should I close it by myself ?

**Pull request merged**